cmd/yggd/main.go

```diff
@@ -358,6 +385,12 @@ func mainAction(c *cli.Context) error {
 		return err
 	}
 
+	fmt.Println("***")
```

Comment:
This looks like a marker line that should not be committed?

Reply:
Oh.